spec/features/users/online_booking/default_spec.rb

```diff
@@ -463,7 +463,7 @@ def add_relative(birth_date: true)
     fill_in("Nom", with: "Lapin")
     fill_in("Date de naissance", with: Date.yesterday) if birth_date
     click_button("Enregistrer")
-    expect(page).to have_content("Mathieu LAPIN")
+    wait_for { User.exists?(first_name: "Mathieu", last_name: "Lapin") }.to be(true)
```

Comment:
peut-être rajouter un commentaire pour rappeler que le expect causait une flaky spec ? 

et éventuellement remettre en dessous de ce `wait_for` le expect initial ? mais vu la description de la flaky avec l’hypothèse que c’est selenium qui perd la ref du DOM probablement que ça ne marcherait pas non plus.

Alors éventuellement rafraichir la page et répeter l’expect ?

Reply:
J'ai essayé de refresh puis vérifier le contenu de la page mais j'ai une erreur "Not attached to an active page". On va se contenter de vérifier la base et d'expliquer en commentaire. :wink: